public/app/features/dashboard/components/GenAI/GenAIHistory.tsx

```diff
@@ -80,7 +79,12 @@ export const GenAIHistory = ({
   };
 
   const onApply = () => {
-    onApplySuggestion(history[currentIndex - 1]);
+    if (isStreamGenerating) {
+      setShouldStop(true);
+      updateHistory(sanitizeReply(reply));
```

Comment:
maybe we should only update history if the reply is not `''` 🤔 ran into this edge case while testing locally